### ISSUE TITLE
overlapping buttons

### ISSUE THREAD

**Issue by ixxakthomas** [Chooloo/koler#316]
in contacts with multiple numbers the call button for each number overlaps.

**Comment by Ilithy** [comment]
Hi, on my phone (Pixel 6 pro - grapheneos-Android 12) there is no such display problem.

_(see screenshot)_
![IMG_20220127_193920](https://user-images.githubusercontent.com/36798218/151423998-331c7ea0-aab1-4965-8c2d-5f30fbfa82b8.jpg)

**Comment by ixxakthomas** [comment]
ooh here the both call buttons are in contact with each other. it would be
nice if there is s gap between them to make them look even better.

**Comment by Ilithy** [comment]
@ixxakthomas Okay, I didn't understand it like that, it's not a bad idea, but not too far apart, otherwise it would enlarge the contact window no?

**Comment by ixxakthomas** [comment]
yeah just a small little gap for aesthetic purposes haha. i love your app
btw and thank you for creating a beautiful one.

**Comment by Ilithy** [comment]
This could be pretty :).

Indeed, thank you @Chooloo for this beautiful application.
